src/node/handler/PadMessageHandler.js

```diff
@@ -1193,48 +1193,46 @@ async function handleClientReady(client, message)
       let cached = historicalAuthorData[author];
 
       // reuse previously created cache of author's data
-      let p = cached ? Promise.resolve(cached) : authorManager.getAuthor(author);
+      const authorInfo = await (cached ? Promise.resolve(cached) : authorManager.getAuthor(author));
```

Comment:
Some lines later authorInfo can be set in a branch. This results in UnhandledPromiseRejectionWarning

Reply:
ups sorry, this has already been fixed :-) thx!